test/Test.purs

```diff
@@ -40,3 +43,11 @@ tests :: TestSuite
 tests = concat (benchmarks <#> asTestSuite)
    <> linkedOutputsSuite linkedOutputs_cases
    <> linkedInputsSuite linkedInputs_cases
+
+benchmarks :: Array BenchSuite
+benchmarks =
+   [ suite loadFile desugar_cases
+   , suite loadFile misc_cases
+   , bwdSuite loadFile bwd_cases
+   , withDatasetSuite loadFile graphics_cases
+   ]
```

Comment:
File missing closing line terminator -- check you have `files.insertFinalNewline` enabled!